code/game/objects/items/storage/belt.dm

```diff
@@ -664,6 +667,7 @@
 	. = ..()
 	var/datum/component/storage/STR = GetComponent(/datum/component/storage)
 	STR.max_items = 24
+	STR.max_combined_w_class = 72
```

Comment:
```suggestion
	STR.max_combined_w_class = 24
```
In the eventual case this can hold larger items it should probably hold less of them